src/kernels/jupyter/connection/jupyterPasswordConnect.ts

```diff
@@ -61,6 +68,12 @@ export class JupyterPasswordConnect implements IJupyterPasswordConnect {
 
                 return value;
             });
+            result.finally(() => {
+                deferred.resolve();
```

Comment:
As soon as the password input is displayed this gets hidden
Then we go back to the multi-select quick pick and the password input gets dismissed